C/CPUInfo/build_tarballs.jl

```diff
@@ -3,21 +3,17 @@
 using BinaryBuilder, Pkg
 
 name = "CPUInfo"
-version = v"0.0.20200122"
+version = v"0.0.20200612"
 
 # Collection of sources required to complete build
 sources = [
-    GitSource("https://github.com/pytorch/cpuinfo.git", "0e6bde92b343c5fbcfe34ecd41abf9515d54b4a7"),
-    DirectorySource("./bundled"),
+    GitSource("https://github.com/pytorch/cpuinfo.git", "63b254577ed77a8004a9be6ac707f3dccc4e1fd9"),
 ]
 
 # Bash recipe for building across all platforms
 script = raw"""
 cd $WORKSPACE/srcdir
 cd cpuinfo
-if [[ $target == *-w64-mingw32 ]]; then
-    atomic_patch -p1 ../patches/lowercase-windows-include.patch
-fi
```

Comment:
So in these 5 months they learnt how to install all the libraries?

Reply:
He he - not quite - they got their capitalization fixed, but the DLL must still be copied manually :-)

Reply:
Oh, right, sorry, I saw the mingw32 `if` and thought it was the other one without even reading